src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -276,10 +306,10 @@ private void execute(ProcBuilder builder) throws KaitaiException {
 			builder.run();
 		} catch (StartupException | TimeoutException | ExternalProcessFailureException e) {
 			throw new KaitaiException(
-				"Fail to execute kaitai command: "
-					+ new String(streamError.toByteArray(), UTF_8)
-					+ new String(streamOutput.toByteArray(), UTF_8)
-				, e
+					"Fail to execute kaitai command: "
+							+ new String(streamError.toByteArray(), UTF_8)
+							+ new String(streamOutput.toByteArray(), UTF_8)
+					, e
```

Comment:
Please revert the indentation changes
```suggestion
				"Fail to execute kaitai command: "
					+ new String(streamError.toByteArray(), UTF_8)
					+ new String(streamOutput.toByteArray(), UTF_8)
				, e
```